x-pack/plugins/canvas/tasks/plugins.js

```diff
@@ -20,17 +21,34 @@ export default function pluginsTasks(gulp, { log, colors }) {
     }
   };
 
+  const copyServerFiles = () => {
+    const serverSourceDir = path.resolve(__dirname, '../canvas_plugin_src/functions/server');
+    const serverBuildDir = path.resolve(__dirname, '../canvas_plugin/functions/server');
+    gulp
```

Comment:
Looks like this is an asynchronous call. Do the `webpack()` calls need to be delayed until gulp is done?

https://github.com/gulpjs/gulp/blob/master/docs/API.md#fn